medusa/subtitles.py

```diff
@@ -811,6 +811,9 @@ def subtitles_download_in_pp():  # pylint: disable=too-many-locals, too-many-bra
 
         run_post_process = False
         for root, _, files in os.walk(app.TV_DOWNLOAD_DIR, topdown=False):
+            # Must ignore files that are being unpacked by NZB client
+            if u'_UNPACK' in root.upper():
```

Comment:
Yes, there it makes sense because root can actually be _UNPACK, but here it is app.TV_DOWNLOAD_DIR, so it will never be _UNPACK. Can't explain it any better.